### PR TITLE
Docs: improved pin security

### PR DESCRIPTION
**Pull request by r4mmer**
### Acceptance Criteria
- Update and QA steps on biometry to also test the new biometry mode.

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: QA.md [hunk 1]
```diff
@@ -62,6 +66,16 @@
         1. Send 1 HTR to your own address (it will ask for the biometry again)
         1. Disable biometry
         1. Lock wallet and it must require you to type your PIN Code.
+    1. Test safe Biometry mode
+        1. Go to unleash and enable the `safe-biometry-mode.rollout` for this wallet.
+        1. Enable biometry (either Fingerprint, FaceID or TouchID depending on the device, it will require your pin)
```
Comment (andreabadesso) [inline QA.md:71]: Biometry will already be enabled at this point, maybe make the last step from the previous section disable the biometry

**Review by andreabadesso — CHANGES_REQUESTED** [review]

File: QA.md (hunk 1 above)
Comment (r4mmer) [reply QA.md:71]: The second last step of the last section is to disable the biometry, so the biometry is not enabled here.
And the first step of the last section is to disable the feature flag, so we need to start with the feature flag enable.

Comment (andreabadesso) [reply QA.md:71]: You're right

**Review by andreabadesso — APPROVED** [review]

File: QA.md [hunk 2]
```diff
@@ -62,6 +66,16 @@
...
+        1. Check that the "Change PIN" option is no longer on the list.
+        1. Send 1 HTR to your own address (it will ask for the biometry instead of pin)
+        1. Close the app and open, it should ask for biometry instead of pin when opening the wallet.
+        1. Send 1 HTR to your own address (it will ask for the biometry again)
+        1. Disable biometry (it will ask for the biometry)
+        1. Check that the "Change PIN" option appears on the list.
+        1. Lock wallet and it must require you to type your PIN Code.
```
Comment (pedroferreira1) [inline QA.md:78]: Does it make sense to test the migration? In this case, we would need to add one more section here with old biometry enabled, then activate in the Unleash and validate if it works fine to load the wallet and send a tx.

What do you think?

**Review by pedroferreira1 — APPROVED** [review]

**Pull request merged**